bpm/bonita-api/bonita-common-api/src/main/java/org/bonitasoft/engine/form/FormMappingTarget.java

```diff
@@ -35,7 +35,12 @@
     LEGACY,
 
     /**
-     * The form mapping is not defined or not necessary
+     * The form mapping is not defined and IS necessary
      */
-    UNDEFINED
+    UNDEFINED,
+
+    /**
+     * The form mapping is not defined and IS NOT necessary
```

Comment:
The form mapping is not defined and IS NOT necessary. This value is automatically set when nothing is specified at design-time. It does not prevent the process to be resolved. It can be used when user tasks are meant to be executed out of a web form context.